codeserver/ubi9-python-3.11/Dockerfile.konflux.cpu

```diff
@@ -8,20 +8,27 @@ WORKDIR /opt/app-root/bin
 # OS Packages needs to be installed as root
 USER 0
 
+# upgrade first to avoid fixable vulnerabilities begin
+RUN dnf -y upgrade --refresh --best --nodocs --noplugins --setopt=install_weak_deps=0 --setopt=keepcache=0 \
+    && dnf clean all -y
+# upgrade first to avoid fixable vulnerabilities end
```

Comment:
I wonder if Konflux Dockerfiles (and maybe any Dockerfiles) should be doing this step, or if they should instead be relying on the base image used in the build being up-to-date?

On that note, I'm surprised that the FROM images aren't specified by digest here, and that digest kept up-to-date by konflux/mintmaker/renovate 🤔